http-server-netty/src/test/groovy/io/micronaut/http/server/netty/configuration/NettyHttpServerConfigurationSpec.groovy

```diff
@@ -57,6 +69,76 @@ class NettyHttpServerConfigurationSpec extends Specification {
         'idle-timeout'       | 'idleTimeout'      | '-1s' | Duration.ofSeconds(-1)
     }
 
+    void "test netty server native transport conditions"() {
+        given:
+        ApplicationContext beanContext = new DefaultApplicationContext("test")
+        beanContext.environment.addPropertySource(PropertySource.of("test",
+              ['micronaut.server.netty.use.native.transport': true]
```

Comment:
The correct configuration option is `micronaut.server.netty.use-native-transport`. I'm surprised this works.

Reply:
Fixed.